src/checkbox/__tests__/common-tests.tsx

```diff
@@ -1,5 +1,7 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
+
+/* eslint-disable jest/no-export */
```

Comment:
This rule fails to detect test helper files. We do not have too many, disabled manually for now